addons/im_livechat/migrations/13.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2020 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+_field_renames = [
+    ('im_livechat.channel', 'im_livechat_channel',
```

Comment:
You need to do more than renaming the field, as this is an attachment field, so there will be attachments with the incorrect `res_field`, so you will lose all the existing ones.

Reply:
but I think that is done in the method below :S

Reply:
You are right, sorry. I didn't remember we included such condition.